### ISSUE TITLE
Could not properly install and use PVinspect

### ISSUE THREAD

**Issue by vohuynhquangnguyen** [ma0ho/pvinspect#7]
I tried to follow the installation provided in the readme:
`conda create -n pvinspect pip python=3.7`
`conda activate pvinspect`
`pip install pvinspect`

Then when I tried to start my dev environment (Jupyter Notebook), it showed an error
`jupyter notebook`
```
ERROR: 'jupyter' is not recognized as an internal or external command,
operable program or batch file.
```
I tried to remedy this error by installing the dev environment via PIP:
`pip install jupyter notebook`
It actually worked and allowed me to enter the dev environment.  Then when I started using it:
`import pvinspect as pv
`
It displayed the following error:
```
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-1-dcf1f302f827> in <module>
----> 1 import pvinspect as pv

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\pvinspect\__init__.py in <module>
----> 1 from . import data, preproc, common, integration
      2 
      3 __all__ = ["data", "preproc", "common", "integration"]
      4 
      5 __version__ = "0.3.1"

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\pvinspect\data\__init__.py in <module>
      1 """Data loading and viewing"""
      2 
----> 3 from . import datasets
      4 from . import io
      5 from . import image

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\pvinspect\data\datasets.py in <module>
      1 """Provides access to demo datasets"""
      2 
----> 3 from .image import (
      4     ModuleImageSequence,
      5     ModuleImage,

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\pvinspect\data\image.py in <module>
      6 import numpy as np
      7 from skimage import io, img_as_uint, img_as_float64, img_as_int
----> 8 from pvinspect.common.transform import Transform
      9 from matplotlib import pyplot as plt
     10 from pathlib import Path

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\pvinspect\common\__init__.py in <module>
      1 """Methods that are used by the other packages"""
      2 
----> 3 from . import transform
      4 from . import util

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\pvinspect\common\transform.py in <module>
      6 from functools import reduce
      7 import cv2
----> 8 from . import util
      9 
     10 

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\pvinspect\common\util.py in <module>
      1 import numpy as np
----> 2 from shapely.geometry import Polygon
      3 from typing import List
      4 from contextlib import contextmanager
      5 import sys, os

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\shapely\geometry\__init__.py in <module>
      2 """
      3 
----> 4 from .base import CAP_STYLE, JOIN_STYLE
      5 from .geo import box, shape, asShape, mapping
      6 from .point import Point, asPoint

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\shapely\geometry\base.py in <module>
     16 
     17 from shapely.affinity import affine_transform
---> 18 from shapely.coords import CoordinateSequence
     19 from shapely.errors import WKBReadingError, WKTReadingError
     20 from shapely.geos import WKBWriter, WKTWriter

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\shapely\coords.py in <module>
      6 from ctypes import byref, c_double, c_uint
      7 
----> 8 from shapely.geos import lgeos
      9 from shapely.topology import Validating
     10 

c:\users\admin\.conda\envs\pvinspect\lib\site-packages\shapely\geos.py in <module>
    143     if os.getenv('CONDA_PREFIX', ''):
    144         # conda package.
--> 145         _lgeos = CDLL(os.path.join(sys.prefix, 'Library', 'bin', 'geos_c.dll'))
    146     else:
    147         try:

c:\users\admin\.conda\envs\pvinspect\lib\ctypes\__init__.py in __init__(self, name, mode, handle, use_errno, use_last_error)
    362 
    363         if handle is None:
--> 364             self._handle = _dlopen(self._name, mode)
    365         else:
    366             self._handle = handle

OSError: [WinError 126] The specified module could not be found
```

**Comment by ma0ho** [comment]
Thanks for reporting this. Could you please try to install `shapely` using conda:

```
conda install shapely
```

and check, whether this resolves the latter issue?

**Comment by vohuynhquangnguyen** [comment]
Hi,

Thank you for your reply.

I tried installing `shapely`, and it did fix the problem. Though it generated another error from `torch`, which I tried to remedy using the same installation command from `conda`:
`conda install pytorch
`
Then, I opened my dev environment and tried importing the module, I received the following warning:
`import pvinspect as pv`
```
c:\users\admin\.conda\envs\pvinspect\lib\site-packages\pvinspect\data\image.py:22: TqdmExperimentalWarning: Using `tqdm.autonotebook.tqdm` in notebook mode. Use `tqdm.tqdm` instead to force console mode (e.g. in jupyter console)
  from tqdm.autonotebook import tqdm
```
Would this warning  affect latter execution of the API?
